metabase/metabase
# Fix relative filters for non-english locale

Fixes https://github.com/metabase/metabase/issues/22361

How to test:
- Change the locale in Account Settings to a non-English one
- Open Products table in a Sample Database
- Make sure it's possible to use relative filters
